templates/assets/cloudformation/elb.yml

```diff
@@ -125,6 +200,9 @@ Resources:
       GroupId: !GetAtt ElbInstanceSecurityGroup.GroupId
   Elb:
     Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+    {{ if not .Loadbalancer.AccessLogs.S3BucketName }}
+    DependsOn: ElbAccessLogsS3BucketPolicy
```

Comment:
Is this DependsOn needed?

Reply:
Unfortunately, I couldn't find a way around it. Cloudformation doesn't seem to automatically depend on it, so there's a race and if the loadbalancer tries to start using the bucket before it exists or has the policy, the loadbalancer fails. It's especially prone to losing that race for existing stacks where the loadbalancer is already running.

Also unfortunate, it seems that cloudformation doesn't have a native way to conditionally "DependOn".